### DISCUSSION TITLE
v1.3.2.0 Release - File Structure Changes

### DISCUSSION THREAD

**Discussion by RadzPrower** [ajohns6/Zelda-3-Launcher#30] [Announcements]
The main snesrev repository recently made changes to its file structure, so I needed to make changes to account for this new structure as file were no longer where they were expected to be.

This should address issue(s) #28/#29 

**Full Changelog**: https://github.com/RadzPrower/Zelda-3-Launcher/compare/v1.3.1.0...v1.3.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/RadzPrower/Zelda-3-Launcher/releases/tag/v1.3.2.0'>v1.3.2.0 Release - File Structure Changes</a>.</em>

**Comment by romik1788** [comment]
bro, during installation it says "Error occurred while downloding pip"

**Reply by RadzPrower** [reply]
Is there a `zelda3.log` which you could share to illustrate the problem? I would review it and sanitize any personal information before posting.

**Reply by romik1788** [reply]
Starting commandline processess...
18.08.2023 19:23:41
----------------------------
Executing via cmd.exe:
 /C .\assets\python.exe .\assets\get-pip.py
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:992)'))': /simple/pip/
WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:992)'))': /simple/pip/
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:992)'))': /simple/pip/
WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:992)'))': /simple/pip/
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:992)'))': /simple/pip/
Could not fetch URL https://pypi.org/simple/pip/: There was a problem confirming the ssl certificate: HTTPSConnectionPool(host='pypi.org', port=443): Max retries exceeded with url: /simple/pip/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:992)'))) - skipping
ERROR: Could not find a version that satisfies the requirement pip (from versions: none)
ERROR: No matching distribution found for pip

**Reply by RadzPrower** [reply]
There's clearly an issue with SSL that is preventing you from downloading pip, but it is not something inherently wrong with the software itself as I was just able to do an install from scratch without issue.

Our locations are obviously different due to the date/time format differences, but I assume this log is from an attempt that was made in the last 15-30 minutes rather than hours ago? If that is the case, either the server was temporarily down and is now available again at the time I ran the installer or there is something blocking the connection on your end. For instance, I am able to click the hyperlink created when you posted that log and it immediately opens up a webpage with a list of pip versions for download even now. I would click that link and see if it opens anything. If it does not, it is an issue with your current connection.
